collections/_patterns/payment.html

```diff
@@ -0,0 +1,78 @@
+---
+title: Payment
+description: Use this pattern when you need to ask users for their payment details
+github-link: true
+toc: true
+---
+
+{% capture payment-pattern %}
+<div class="sgds form-group mb-3">
+    <label for="nameOnCard" class="form-label">Name on Card</label>
+    <input type="text" class="form-control" id="name" autocomplete="nameOnCard">
+</div>
+<div class="sgds form-group mb-3">
+    <label for="cardNumber" class="form-label">Card Number</label>
+    <div id="cardNumberText" class="form-text">Enter the 16-digit number on the card</div>
+    <input type="text" class="form-control" id="cardNumber" autocomplete="cardNumber" aria-describedby="expiryDateText">
+</div>
+<div class="row">
+  <div class="sgds col form-group mb-3">
+    <label for="expiryDate" class="form-label">Expiry Date</label>
+    <div id="expiryDateText" class="form-text">MM/YY</div>
+    <input type="text" class="form-control" id="expiryDate" autocomplete="expiryDate" aria-describedby="expiryDateText">
```

Comment:
could you try to implement this for MM/YY 
- only allow digits 
- on typing two digit , autocomplete with a "/" 
- type the next two digit 

if need be write javascript